### DISCUSSION TITLE
Banned IP Addresses

### DISCUSSION THREAD

**Discussion by sysbitnet** [AzuraCast/AzuraCast#7607] [Ideas]
When i tested i found this IP 34.140.250.78 and when i tried to block it like on the screenshot image
![Screenshot 2024-12-05 204028](https://github.com/user-attachments/assets/794b283c-2bad-4e2f-b5b5-b37fb8d0574a)

The action wasn't successful, Here’s a simple example of IP addresses or groups in CIDR format listed one per line:
192.168.1.0/24
10.0.0.0/8
172.16.0.0/12
203.0.113.0/24
0.0.0.0/0
::/0

Of course, i **Restarted Broadcasting** how can I be sure it is all working, but the IP shows again.

After that, i put just this IP on the line how can block.

But it would been nice if i could block the IP range if needed.

Powered by [AzuraCast](https://www.azuracast.com/) • v0.20.4 Stable • Docker • PHP 8.3

**Comment by bombelman** [comment]
this does not work from AzuraCast, already reported this, no reply really.
a workaround was posted there:

https://github.com/AzuraCast/AzuraCast/issues/7194
